<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Gui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="createQuestions">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ceate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startSrever">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Sever</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addQuestoin">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delQuestion">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finishQuestions">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>392</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
   <widget class="QListWidget" name="questionsList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>201</y>
      <width>441</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
